mitodl/mit-open
# Shanbady/endpoint to retrieve session data

### What are the relevant tickets?

Closes #614 


### Description (What does it do?)
This adds fields that the frontend needs to /api/v0/user/me
In addition to that, it also replicates /api/v0/user/me in the v1 api so that open api generates clients for it. The v1 api is marked as experimental.

### How can this be tested?
1. checkout this branch
2. visit the v0 user api (/api/v0/users/me) which should now contain new fields first_name, last_name, is_article_editor, and is_learning_path_editor
3. visit the v1 api and see that it matches the v0 api  /api/v1/users/me
4. see that the new fields and new api route exist in the open api spec
